packages/clay-dropdown/src/ClayDropdown.soy

```diff
@@ -64,5 +65,6 @@
 		{param triggerClasses: $triggerClasses /}
 		{param triggerSize: $triggerSize /}
 		{param triggerTitle: $triggerTitle /}
+		{param usePortal: $usePortal ?: true /}
```

Comment:
hey @carloslancha can you do the same for `ClayActionsDropdown` and `ClayCreationMenuDropdown`?